recaf-ui/src/main/java/me/coley/recaf/ui/control/config/ConfigBoolean.java

```diff
@@ -23,7 +26,15 @@ public class ConfigBoolean extends CheckBox {
 	 */
 	public ConfigBoolean(ConfigContainer instance, Field field, ObservableValue<String> text) {
 		textProperty().bind(text);
-		setSelected(ReflectUtil.quietGet(instance, field));
-		selectedProperty().addListener((observable, old, current) -> ReflectUtil.quietSet(instance, field, current));
+		if (ObservableBooleanValue.class.isAssignableFrom(field.getType())) {
+			var value = (ObservableBooleanValue) ReflectUtil.quietGet(instance, field);
+			if (value instanceof BooleanProperty)
```

Comment:
```suggestion
			if (value instanceof BooleanProperty) {
```
Why not using curly brackets here?

Reply:
XD

Reply:
Rly xD